CHANGELOG.md

```diff
@@ -271,6 +273,9 @@ Indexes iterators names has been shortened to `Iter`, `Keys` and `Values`. (#162
 
 - Added maintenance command `restart-migration` to restart migration script. (#1728)
 
+- Added `with_default_instance` and `with_default_service` to start service
```

Comment:
Nit: I'd qualify "Added `with_default_instance` and `with_default_service` to `NodeBuilder` in order to start service instances..."